src/components/PaymentLog.vue

```diff
@@ -514,10 +550,24 @@ export default {
         this.$displayErrorMessage(err);
       }
     },
+    async getDiscounts() {
+      try {
+        const fetchDiscounts = await get(
+          this.apiUrls.discounts
+        );
+        this.discounts = fetchDiscounts.data.data.data
+        this.discounts = this.discounts.filter((item)=> {
+          return item.id == 6 || item.id == 1});
```

Comment:
what ids are these?

Reply:
new sales & renewal. But I checked now, you have changed the ids

Reply:
I have removed it